internal/driver/daemon.go

```diff
@@ -6,13 +6,46 @@ import (
 	"net/http"
 	"strings"
 
+	"github.com/ory/analytics-go/v4"
+	"github.com/ory/x/healthx"
+	"github.com/ory/x/metricsx"
+	"github.com/spf13/cobra"
+
+	"github.com/ory/keto/internal/driver/config"
+
 	"github.com/ory/graceful"
 	"github.com/pkg/errors"
 	"github.com/soheilhy/cmux"
 	"golang.org/x/sync/errgroup"
 	"google.golang.org/grpc"
 )
 
+func (r *RegistryDefault) EnableSqa(cmd *cobra.Command) {
+	r.sqaService = metricsx.New(
+		cmd,
+		r.Logger(),
+		r.Config().Source(),
+		&metricsx.Options{
+			Service:       "ory-keto",
+			ClusterID:     metricsx.Hash(r.Config().DSN()),
```

Comment:
Are there other values which are kinda unique and stable?

Reply:
No not really, maybe the `ListenOn` but that will probably not be very unique.

Reply:
Ok then this is enough for now